### ISSUE TITLE
BOX index.js does not include all of your functions

### ISSUE THREAD

**Issue by airburst** [FormulaPages/box#4]
Hi Peter,

Fantastic work with formulapages thanks.

Did you intend formula-box to pull all functions in?  I noticed that LOOKUP, HLOOKUP and VLOOKUP are all present in src, but are not included in the index.js file and so are not exposed by box as methods.

Regards

**Comment by petermoresi** [comment]
First, they should be in the index.js. This is a mistake. Thanks for reporting it.

Second, thank you for the appreciation. I've been working on this library for over a year now and so far it has gone larger unnoticed. I've learned so much doing this work and it has been very useful to me.

Third, I'm moving this project to a new home https://github.com/FunctionFoundry/functionfoundry. The issue with LOOKUP, HLOOKUP and VLOOKUP are fixed there and I added test cases for the index.js since this has bitten me multiple times now.

**Comment by airburst** [comment]
No problem - thanks.  

I built a prototype for a business application around 3 years ago and one aspect of it was crying out for JS versions of Excel functions to run calculations in the client, but nothing existed at the time and I only wrote a handful to prove the concept.

Your work is excellent and ES6 makes the modules much cleaner.

Will function foundry be a named module on NPM, or will you stick with formula-box?

Cheers
